client/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Name-Request",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bcgov/namerequest.git"
+  },
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache License, Version 2.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
```

Comment:
We've done this with this script so it's easy to run unit tests for a specific component:
`"test:unit": "vue-cli-service test:unit --testPathPattern=.",`
(Just replace . with the component file name.)

Reply:
thats handy thank you.